collector/shared/version.go

```diff
@@ -1,42 +0,0 @@
-package collector_shared
```

Comment:
It turns out I already preserved the the version checking function in [shared/connection.go](https://github.com/timvaillancourt/prometheus_mongodb_exporter/blob/upstream_updates_bugfixes/shared/connection.go#L36-43), so the only thing left was the comparison function. I've added it to shared/utils.go (a better place for it anyways) in this commit:

https://github.com/timvaillancourt/prometheus_mongodb_exporter/commit/37ec32bf5e157010df5c3493fbb79762752074ea